components/datepicker/modules/calendar.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div class="d-datepicker__calendar">
+    {{ weekDays }}
+    <div
+      v-for="(week, indexWeek) in calendarDays"
+      :key="indexWeek"
+    >
+      {{ indexWeek }}
+      <button
+        v-for="(day, indexDays) in week.days"
+        :key="indexWeek + indexDays"
+        type="button"
+        :aria-label="day.text"
+        @click="selectDay(day)"
+        @keydown="$emit('select-date', day.value)"
```

Comment:
Yes this is right, this should be improved in the respective ticket since this is a stub only but I can do it now.

And yes it should be triggered on `click` and `keydown.space` or even `.enter`